easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.5.2-iimpi-2020a.eb

```diff
@@ -13,6 +13,10 @@ source_urls = ['https://github.com/Unidata/netcdf-fortran/archive/']
 sources = ['v%(version)s.tar.gz']
 checksums = ['0b05c629c70d6d224a3be28699c066bfdfeae477aea211fbf034d973a8309b49']
 
+builddependencies = [
+     ('M4', '1.4.18'),
```

Comment:
```suggestion
    ('M4', '1.4.18'),
```

Reply:
On the other file, I could edit from github, but this one I can't